pkg/cluster/storageSuffix.go

```diff
@@ -0,0 +1,36 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+)
+
+func openShiftClusterDocumentMutatorFn(doc *api.OpenShiftClusterDocument) error {
+	const ImageRegistry = "imageregistry"
+
+	if doc.OpenShiftCluster.Properties.StorageSuffix == "" {
+		var err error
+
+		doc.OpenShiftCluster.Properties.StorageSuffix, err = randomLowerCaseAlphanumericStringWithNoVowels(5)
```

Comment:
Is it necessary to have it this way? wouldn't `:=` be sufficient here ?

Reply:
In that case no because the result is directly assigned to m.doc and it's not a new variable, so -> Error "expected identifier on left side of :="
But using a temp variable can be done, so done. Thanks for the suggestion 😄